packages/grid/src/NumericColumn.tsx

```diff
@@ -50,6 +50,10 @@ export function NumericCellEditor<T>(props: NumericEditorProps<T>) {
   const onChange: ChangeEventHandler<HTMLInputElement> = (e) => {
     setEditorText(e.target.value);
   };
+  
+  const onBlur: ChangeEventHandler<HTMLInputElement> = (e) => {
```

Comment:
This should be `FocusEventHandler<HTMLInputElement>`

Reply:
Yeah Sure will Update it Thanks a lot